packages/ni-redirects/src/documents.js

```diff
@@ -0,0 +1,48 @@
+const documentPrefix = '/wp-content/ipc/uploads/projects/';
+
+// document paths are in the form /wp-content/ipc/uploads/projects/{caseRef}/{docRef}-{fileName}
+const documentRegex = new RegExp(
+	/^\/wp-content\/ipc\/uploads\/projects\/(?<caseRef>[A-Za-z0-9]*)\/([A-Za-z0-9]*)-(?<docRef>[A-Za-z0-9]*)-(?<fileName>.*)$/
```

Comment:
Will this match if URL is:
https://nipreprod.planninginspectorate.gov.uk/wp-content/ipc/uploads/projects/EN010120/EN010120-000357........

Do we need to add the prefix or ignore it?

Reply:
The hostname part? We only pass through the path:

https://github.com/Planning-Inspectorate/applications-service/pull/1035/files#diff-b4779657cc4bd4d5e5e2267db7ecc3f7ad3a89715de2ae3de133c4dac4ad385dR24-R25